### PR TITLE
[chore] close page before closing server

### PR DESCRIPTION
**Pull request by benmccann**
It seems the server may not be closing. The docs for `server.close` say "the server is finally closed when all connections are ended". Try doing `context.close()` first to see if it kills any dangling connections.

### CODE REVIEW

**Review by dummdidumm — APPROVED** [review]

File: packages/kit/test/apps/basics/test/server.test.js [hunk 1]
```diff
@@ -330,7 +330,7 @@ test.describe('Load', () => {
 			await page.goto(`/load/fetch-origin-external?port=${port}`);
 			expect(await page.textContent('h1')).toBe(`origin: ${new URL(baseURL).origin}`);
 		} finally {
-			await page.close();
+			await context.close();
 			close();
```
Comment (dominikg) [inline packages/kit/test/apps/basics/test/server.test.js:334]: is this line missing an await?

**Comment by dominikg** [comment]
according to the playwright docs, context is created fresh for each test and they never mention that one has to manually close it. Would it be something to bring up with them? Feels really strange that we'd have to do it

File: packages/kit/test/apps/basics/test/server.test.js (hunk 1 above)
Comment (dummdidumm) [reply packages/kit/test/apps/basics/test/server.test.js:334]: mhm good question - @benmccann does this make a difference? (same happens in another test that was changed here)

Comment (benmccann) [reply packages/kit/test/apps/basics/test/server.test.js:334]: I don't know. It's an interesting question. We should be consistent one way or the other though: https://github.com/sveltejs/kit/pull/8447

**Comment by benmccann** [comment]
I'm guessing it'd be automatically closed after the test is done running, but here I wanted to do it before we tried to close the server to ensure that there's no outstanding connections to the server as it seems that may make the server hang

**Pull request merged**